packages/dialtone-vue3/components/rich_text_editor/rich_text_editor_default.story.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <dt-rich-text-editor
-    v-model="$attrs.modelValue"
+    v-model="value"
```

Comment:
why not using `$attrs.modelValue` directly?

Reply:
`$attrs.modelValue` is readonly, and v-model tries to change it. It is common for us to do this with v-model. See `dropdown_default.story.vue` for example.